### DISCUSSION TITLE
What is api?

### DISCUSSION THREAD

**Discussion by FaysalAhmed155** [community/community#142452] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

 I'm new at programming. I don't know what is a API. Can anyone tell me about this?

**Comment by webdevmahdi** [comment]
The full form of API is Application Programming Interface. API is mainly used to communicate with the back-end server from the front-end. API can hold a lot of information with itself.

**Reply by rov1dev** [reply]
web Assistant

**Comment by ebndev** [comment]
Hi @FaysalAhmed155 & @webdevmahdi, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. 

Thanks for understanding and we hope to see you participate in other discussions in the future!
